app/models/photo.rb

```diff
@@ -96,20 +96,22 @@ def processed?
   end
 
   def update_remote_path
-    unless self.unprocessed_image.url.match(/^https?:\/\//)
-      remote_path = "#{AppConfig.pod_uri.to_s.chomp("/")}#{self.unprocessed_image.url}"
-    else
-      remote_path = self.unprocessed_image.url
-    end
+    remote_path = if unprocessed_image.url.match(%r{^https?://})
+                    unprocessed_image.url
+                  else
+                    "#{AppConfig.pod_uri.to_s.chomp('/')}#{unprocessed_image.url}"
+                  end
 
-    name_start = remote_path.rindex '/'
+    name_start = remote_path.rindex "/"
     self.remote_photo_path = "#{remote_path.slice(0, name_start)}/"
     self.remote_photo_name = remote_path.slice(name_start + 1, remote_path.length)
   end
 
-  def url(name = nil)
-    if remote_photo_path
-      name = name.to_s + '_' if name
+  def url(name=nil)
+    # During migration in bad cases this might be happen
```

Comment:
Again, these two lines could be clearer, but context would be helpful.

Reply:
A message to a coder. The if clause the next line was changed, because in rare cases a photo can not be processed. Then the stream will not load. Diaspora stops (crashes). Would be nice to use clear words for this.

Reply:
How about 'If a photo cannot be processed during migration, the migration will fail and the stream will stop loading. If this happens, don't show the photos' ???